Application/Commands/Enquiry/Respond/AddEnquiryResponseCommand.cs

```diff
@@ -163,95 +144,123 @@ public async Task<ResponseConfirmationModel> Handle(AddEnquiryResponseCommand re
         {
-            { EnquiryTuitionPartnerNameKey, input.TpName! },
-            { EnquiryLadNameKey, input.LocalAreaDistrict! },
-            { EnquiryKeyStageAndSubjects, input.EnquiryKeyStageSubjects! },
-            { EnquiryResponseKeyStageAndSubjects, input.EnquiryResponseKeyStageSubjects! },
-            { EnquiryTuitionTypeKey, input.EnquiryTuitionType! },
-            { EnquiryResponseTuitionTypeKey, input.EnquiryResponseTuitionType! },
-            { EnquiryTuitionPlanKey, input.EnquiryTuitionPlan! },
-            { EnquiryResponseTuitionPlanKey, input.EnquiryResponseTuitionPlan! },
-            { EnquirySENDSupportKey, input.EnquirySendSupport! },
-            { EnquiryResponseSENDSupportKey, input.EnquiryResponseSendSupport! },
-            { EnquiryAdditionalInformationKey, input.EnquiryAdditionalInformation! },
-            { EnquiryResponseAdditionalInformationKey, input.EnquiryResponseAdditionalInformation! }
+            { EnquiryTuitionPartnerNameKey, _tpName! },
+            { EnquiryLadNameKey, request.Data.LocalAuthorityDistrict },
+            { EnquiryKeyStageAndSubjects, string.Join(Environment.NewLine, request.Data.EnquiryKeyStageSubjects!) },
+            { EnquiryResponseKeyStageAndSubjects, request.Data.KeyStageAndSubjectsText.EscapeNotifyText(true)! },
+            { EnquiryTuitionTypeKey, request.Data.EnquiryTuitionType },
+            { EnquiryResponseTuitionTypeKey, request.Data.TuitionTypeText.EscapeNotifyText(true)! },
+            { EnquiryTuitionPlanKey, request.Data.EnquiryTutoringLogistics.EscapeNotifyText()! },
+            { EnquiryResponseTuitionPlanKey, request.Data.TutoringLogisticsText.EscapeNotifyText(true)! },
+            { EnquirySENDSupportKey, request.Data.EnquirySENDRequirements.EscapeNotifyText() ?? StringConstants.NotSpecified },
+            { EnquiryResponseSENDSupportKey, request.Data.SENDRequirementsText.EscapeNotifyText(true) ?? StringConstants.NotSpecified },
+            { EnquiryAdditionalInformationKey, request.Data.EnquiryAdditionalInformation.EscapeNotifyText() ?? StringConstants.NotSpecified },
+            { EnquiryResponseAdditionalInformationKey, request.Data.AdditionalInformationText.EscapeNotifyText(true) ?? StringConstants.NotSpecified }
         };
 
-        return personalisation;
+        personalisation.AddDefaultEnquiryPersonalisation(_enquiryReferenceNumber!, request.Data!.BaseServiceUrl!, _createdDateTime!);
+
+        return personalisation.Select(x => new EmailPersonalisationLog
+        {
+            Key = x.Key,
+            Value = x.Value.ToString()
+        }).ToList();
     }
 
+    private async Task ProcessEmailsAsync(EnquiryResponse enquiryResponse)
+    {
+        try
+        {
+            await _processEmailsService.SendEmailAsync(enquiryResponse.EnquirerResponseEmailLogId);
+        }
+        catch { } //We suppress the exceptions here since we want the user to get the confirmation page, errors are logged in the services
```

Comment:
Seeing the following error on my IDE "Nested blocks of code should not be left empty".

We should put the comment inside of the catch block.

For example:
```

catch
{
    // We suppress the exceptions here since we want the user to get the confirmation page, errors are logged in the services
} 
```

Reply:
As above